cmd/podman/common.go

```diff
@@ -320,6 +320,14 @@ var createFlags = []cli.Flag{
 		Usage: "Tells podman how to handle the builtin image volumes. The options are: 'bind', 'tmpfs', or 'ignore' (default 'bind')",
 		Value: "bind",
 	},
+	cli.BoolFlag{
+		Name:  "init",
+		Usage: "Run an init binary inside the container that forwards signals and reaps processes",
```

Comment:
I hate calling this an init binary. It is really a reaper.  But this is probably what Docker calls it.  :^(

Does it forward signals to all of the processes within the container?  How does it keep track?

Reply:
I basically copied the Docker wording and signal forwarding is part of what's expected from the init binary.